x-pack/test/api_integration/deployment_agnostic/default_configs/stateful.config.base.ts

```diff
@@ -98,6 +105,17 @@ export function createStatefulTestConfig<T extends DeploymentAgnosticCommonServi
         ...xPackAPITestsConfig.get('kbnTestServer'),
         serverArgs: [
           ...xPackAPITestsConfig.get('kbnTestServer.serverArgs'),
+          // if the config is run locally, explicitly enable mock-idp-plugin for UI role selector
+          ...(isRunOnCI ? [] : ['--mockIdpPlugin.enabled=true']),
```

Comment:
this plugin is not part of CI build and FTR config failed to start: 
```
2024-09-10 16:25:03 CEST	 proc [kibana] \u001b[37m\u001b[41m FATAL \u001b[49m\u001b[39m Error: Unknown configuration key(s): "mockIdpPlugin.enabled". Check for spelling errors and ensure that expected plugins are installed.
2024-09-10 16:25:03 CEST	 proc [kibana]
2024-09-10 16:25:03 CEST	ERROR Error: [kibana] exited without matching pattern: /Kibana is now available/
2024-09-10 16:25:03 CEST	          at createFailError (dev_cli_errors.ts:28:24)
```

There is no such problem on serverless, because plugin is enabled in Kibana by default and we don't explicitly start it in FTR config. Since we can't and don't want to do the same for stateful, we disabled it by default and pass here only when config is executed locally